### DISCUSSION TITLE
下拉选择器在滚动时定位问题

### DISCUSSION THREAD

**Discussion by six-ben** [rsuite/rsuite#2589] [General]
需求描述：像 DatePicker 组件默认渲染到 body 上的，也就是相对于根元素定位。当项目布局的 content 是自定义的滚动时，会出现页面内容滚动，但是选择器不滚动的情况。

建议提供一个可以自定义相对于父节点定位的属性，例如 antd 的 getPopupContainer 属性

getPopupContainer 功能：https://codesandbox.io/s/4j168r7jw0

**Comment by simonguo** [comment]
rsuite 所有的 Picker 组件都有一个 `container` 属性。
https://rsuitejs.com/components/date-picker/#props
